src/blog/2023/03/comparinig-dashboards.md

```diff
@@ -0,0 +1,179 @@
+
+<!--more-->
+
+## Which dashboards am I going to consider?
+
+At this time I believe there are 3 main dashboards worth considering based on their downloads per week and active development. In no particular order, they are [Dashboard](https://flows.nodered.org/node/node-red-dashboard), [uibuilder](https://flows.nodered.org/node/node-red-contrib-uibuilder), and [FlexDash](https://flows.nodered.org/node/@flexdash/node-red-fd-corewidgets).
+
+It's not to say that there are not other options, I am focusing on the dashboards I believe you are most likely to use day to day.
+
+I'd like to take this opportunity to thank the project leads for each of the three dashboards for responding to me and providing their take on the current state and future development of each. Where possible I have quoted their words, either from their messages to me or from the projects' documentation. Thanks to [Dave](https://github.com/dceejay), [Julian](https://github.com/TotallyInformation), and [Thorsten](https://github.com/tve) for their replies as well as all the work they've put into these great projects!
+
+## Methodology
+
+To compare these dashboards, I am going to consider each of them based on the following factors:
+
+- [How easy is it to install?](#how-easy-is-it-to-install%3F)
+
+- [How easy is it to get your first demo dashboard running?](#how-easy-is-it-to-get-your-first-demo-dashboard-running%3F)
+
+- [How extensive is the collection of UI elements?](#how-extensive-is-the-collection-of-ui-elements%3F)
+
+- [How good is the support and documentation?](#how-good-is-the-support-and-documentation%3F)
+
+- [How 'cloud native' is the dashboard?](#how-'cloud-native'-is-the-dashboard%3F)
+
+- [How active is each project's development?](#how-active-is-each-project's-development%3F)
+
+- [What are the future development plans?](#what-are-the-future-development-plans%3F)
+
+I am assuming the user is a low-code developer. They may have limited experience with coding but they are most comfortable working in visual interfaces.
+
+I am also assuming that the user has been recommended to search for each of these dashboards by name. It's a little unfair to say that Dashboard is inherently better because its name is an obvious search term for 'dashboards'.
+
+So, that's the methodology, let's get on with looking at the strengths of each project.
+
+## How easy is it to install?
+
+### uibuilder - 1st place
+
+A search on [DuckDuckGo for the term 'node red ui builder'](https://duckduckgo.com/?q=node+red+ui+builder&t=h_&ia=web) returns the correct custom node. When searching for the custom node in the palette manager there is only one result, this is great as users are very likely to install what they were searching for. Once you've found the correct custom node, the installation takes just a few moments using the palette manager.
```

Comment:
I am curious why you are using DuckDuckGO for search. A very large majority of people use Google for search?

Reply:
DDG gives the same results to all users where as Google search is very inconsistent. I feel using DDG is a more valid way to understand current search rankings but I don't mind changing it.